src/Nest/Document/Multiple/BulkAll/BulkAllRequest.cs

```diff
@@ -175,8 +185,19 @@ public BulkAllDescriptor(IEnumerable<T> documents)
 		/// <inheritdoc />
 		public BulkAllDescriptor<T> Pipeline(string pipeline) => Assign(p => p.Pipeline = pipeline);
 
-		/// <inheritdoc />
-		public BulkAllDescriptor<T> BufferToBulk(Action<BulkDescriptor, IList<T>> modifier) => Assign(p => p.BufferToBulk = modifier);
-
+		/// <inheritdoc />
+		public BulkAllDescriptor<T> BufferToBulk(Action<BulkDescriptor, IList<T>> modifier) => Assign(p => p.BufferToBulk = modifier);
+
+		/// <summary>
+		/// Simple back pressure implementation that makes sure the minimum max concurrency between producer and consumer
+		/// is not amplified by the greedier of the two by more then a given back pressure factor
+		/// When set each scroll request will additionally wait on <see cref="ProducerConsumerBackPressure.WaitAsync"/> as well as
+		/// <see cref="MaxDegreeOfParallelism"/> if set. Not that the consumer has to call <see cref="ProducerConsumerBackPressure.Release"/>
+		/// on the same instance every time it is done.
+		/// <param name="maxConcurrency">The minimum maximum concurrency which would be the bottleneck of the producer consumer pipeline</param>
+		/// <param name="backPressureFactor">The maximum amplification back pressure of the greedier part of the producer consumer pipeline</param>
```

Comment:
I think we should point out what the default backPressureFactor is if unspecified.